test/metabase/lib/metadata/calculation_test.cljc

```diff
@@ -129,8 +129,7 @@
                                  :source-card (:id (lib.tu/mock-cards :orders))}]}
           own-fields (for [field (lib.metadata/fields lib.tu/metadata-provider-with-mock-cards (meta/id :orders))]
                        (-> field
-                           (assoc :lib/source :source/card)
-                           (dissoc :id :table-id)))]
```

Comment:
I'm assuming this is intended to simulate Real Life, so I'm unsure we can just remove this. But I've found your Slack message, and now I tend to think that we should test both possibilities. If there are no `id`s, the names should provide unique references, I guess.

Reply:
Added tests where id and table-id are not present